python/ray/serve/metrics.py

```diff
@@ -0,0 +1,34 @@
+from ray.util import metrics
+from typing import Tuple, Optional, Dict
+
+REPLICA_TAG = "N/A"
+DEPLOYMENT = "N/A"
```

Comment:
let's just centralize all of this global metadata in one place (`context.py`) and pull from there

no reason not to add route and other info there while we're at it